runtime/src/proxy.rs

```diff
@@ -65,9 +84,10 @@ impl InstanceFilter<RuntimeCall> for ProxyType {
                 RuntimeCall::SettlementGroth16Pallet(..) |
                 RuntimeCall::SettlementRisc0Pallet(..) |
                 RuntimeCall::SettlementUltraplonkPallet(..) |
-                RuntimeCall::SettlementPlonky2Pallet(..) |
-                RuntimeCall::SettlementProofOfSqlPallet(..)
-            ),
+                RuntimeCall::SettlementProofOfSqlPallet(..) |
+                RuntimeCall::SettlementPlonky2Pallet(..)
```

Comment:
Maybe we could add a one line comment to remind that also `is_a_submit_proof_extrinsic` should be updated when adding new verifiers here?